cumulusci/tasks/salesforce/tests/test_network_member_group.py

```diff
@@ -0,0 +1,594 @@
+import unittest
+from unittest.mock import Mock, call
+from cumulusci.tasks.salesforce.tests.util import create_task
+from cumulusci.tasks.salesforce.network_member_group import CreateNetworkMemberGroups
+from cumulusci.core.exceptions import SalesforceException, CumulusCIException
+
+
+class TestCreateNetworkMemberGroups(unittest.TestCase):
+    """
+    Unit tests cumulusci.tasks.salesforce.network_member_group.CreateNetworkMemberGroups.
+    """
+
+    def test_get_network_id__no_network_found(self):
+        network_name = "network_name"
+
+        task = create_task(CreateNetworkMemberGroups, {"network_name": network_name})
+
+        task.sf = Mock()
+        task.sf.query = Mock(return_value={"records": []})
```

Comment:
Tip: all attributes of Mocks are also Mocks, so you can here even just do

```suggestion
        task.sf.query.return_value = {"records": []}
```